migrations/5-create-topping.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('topping', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      name: {
+        type: Sequelize.STRING
+      },
+      price_extra: {
```

Comment:
Need to standardize on either under_score, or camelCase. You use both in the same model, that's going to make your life harder later.